instrumentation/rack/lib/opentelemetry/instrumentation/rack/instrumentation.rb

```diff
@@ -29,6 +29,7 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         option :retain_middleware_names,  default: false, validate: :boolean
         option :untraced_endpoints,       default: [],    validate: :array
         option :url_quantization,         default: nil,   validate: :callable
+        option :filtered_requests,        default: nil,   validate: :callable
```

Comment:
For consistency with `untraced_endpoints`, I think we should call this `untraced_requests`:
```suggestion
        option :untraced_requests,        default: nil,   validate: :callable
```